src/components/Users.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from "react";
+import {
+  CircularProgress,
+  Container,
+  List,
+  ListItem,
+  ListItemText,
+  TextField,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { routes } from "../routes/routes";
+import { searchUsers } from "../utils/utils";
+import { useGetHandler } from "../network/useQueryClient";
+import {keys} from '../network/keys'
+import {urls} from '../network/urls'
+
+const Users = () => {
+
+  const [query, setQuery] = useState("");
+  const [users, setUsers] = useState([])
+
+  const navigate = useNavigate();
+
+  const {data, status} = useGetHandler(keys.USERS, urls.USERS)   
```

Comment:
instead os status, use isLoading, it's a boolean so you wouldn't have to compare it with string literal.
```suggestion
  const {data, isLoading} = useGetHandler(keys.USERS, urls.USERS)   
```